### PR TITLE
first cut at sqlite support

### PR DESCRIPTION
**Pull request by jtomson**
Very cool project! This PR addresses issue #17, adding basic sqlite support

### CODE REVIEW

File: cli/args.js [hunk 1]
```diff
@@ -7,6 +7,11 @@ module.exports = require('yargs')
         describe: 'Database name',
         type: 'string'
     })
+    .option('filename', {
```
Comment (rexxars) [inline cli/args.js]: Could you make this `db-filename` instead (accessible in code as `dbFilename`)?

**Comment by rexxars** [comment]
This looks great! Could you change from double quotes (`"`) to single quotes (`'`) for strings? That would match our coding style :+1: 

Thanks for your contribution!

**Comment by jtomson** [comment]
>  Could you change from double quotes (") to single quotes (') for strings?

Whoops! Done!

File: cli/args.js (hunk 1 above)
Comment (jtomson) [inline cli/args.js]: absolutely - done!

**Comment by rexxars** [comment]
This is great - it does have a few flaws though. I tried it on the chinook (https://chinookdatabase.codeplex.com/) database, and it has a hard time recognizing some types:

- Types are not lowercased, so `INTEGER` is not recognized
- There is apparently an `nvarchar` type that we should be able to map directly to a string, but it has a size specifier that needs to be removed (`nvarchar(36)`, for instance)

There is also several lint errors which would be nice to get rid of. If you're still keen on getting this merged, could you download the chinook example database and try to get this running on that?

Make sure to run `npm test` to run the linter once you've got it working :-)

**Comment by jtomson** [comment]
sure thing - the `chinook` database seems to parse without issue now and `npm test` reports no linter issues

sql-to-graphql_playground_and_first_cut_at_sqlite_support_by_jtomson_ _pull_request__36_ _vaffel_sql-to-graphql_and_s2g_ _-bash_ _281x74

**Comment by rexxars** [comment]
Sweet! Thanks a lot!

**Comment by jtomson** [comment]
Cheers!

**Pull request merged**